plugins/git-build.js

```diff
@@ -212,7 +215,7 @@ function buildConsumer(config, cimpler, repoPath) {
          if (!config.logs || !config.logs.path)
             return null;
 
-         var logFilename = (inBuild.branch || 'HEAD') + "--" +
+         var logFilename = (inBuild.branch.replace(/[^0-9a-zA-Z]+/g, '-') || 'HEAD') + "--" +
```

Comment:
If you're okay with `_` being included, you could use
```suggestion
         var logFilename = (inBuild.branch.replace(/\W+/g, '-') || 'HEAD') + "--" +
```

Reply:
I prefer the slightly more explicit and slightly more limited version. Combining `_` and `-` in one name is rather annoying :-)